.gitignore

```diff
@@ -0,0 +1,2 @@
+doc/
```

Comment:
Doc directory should not be ignored by git.

Reply:
Its generated for local docs, no ?

Reply:
No, only doc/html is I believe.